src/core/Akka.FSharp/FsApi.fs

```diff
@@ -481,6 +480,74 @@ let asyncReceive (i : Inbox) : Async<'Message option> =
+/// When an actor refered by subject dies, a watcher should receive a <see cref="Terminated"/> message.
+/// </summary>
+let monitor (subject: ActorRef) (watcher: ICanWatch) : ActorRef = watcher.Watch subject
+
+/// <summary>
+/// Orders a <paramref name="watcher"/> to stop monitoring an actor refered by provided <paramref name="subject"/>.
+/// </summary>
+let demonitor (subject: ActorRef) (watcher: ICanWatch) : ActorRef = watcher.Unwatch subject
+
+/// <summary>
+/// Subscribes an actor reference to target channel of the provided event stream.
+/// </summary>
+let sub (channel: System.Type) (ref: ActorRef) (eventStream: Akka.Event.EventStream) : bool = eventStream.Subscribe(ref, channel)
+
+/// <summary>
+/// Unubscribes an actor reference from target channel of the provided event stream.
+/// </summary>
+let unsub (channel: System.Type) (ref: ActorRef) (eventStream: Akka.Event.EventStream) : bool = eventStream.Unsubscribe(ref, channel)
+
+/// <summary>
+/// Publishes an event on the provided event stream. Event channel is resolved from event's type.
+/// </summary>
+let pub (event: 'Event) (eventStream: Akka.Event.EventStream) : unit = eventStream.Publish event
+
+let private taskContinuation (task: System.Threading.Tasks.Task) : unit =
+    match task.IsFaulted with
+    | true -> raise task.Exception
+    | _ -> ()
+
+/// <summary>
+/// Schedules a function to be invoked repeatedly in the provided time intervals. 
+/// </summary>
+/// <param name="after">Initial delay to first function call.</param>
+/// <param name="every">Interval.</param>
+/// <param name="fn">Function called by the scheduler.</param>
+/// <param name="scheduler"></param>
+let schedule (after: TimeSpan) (every: TimeSpan) (fn: unit -> unit) (scheduler: Scheduler): Async<unit> =
+    let action = Action fn
+    Async.AwaitTask (scheduler.Schedule(after, every, action).ContinueWith taskContinuation)
+    
+/// <summary>
+/// Schedules a single function call using specified sheduler.
+/// </summary>
+/// <param name="after">Delay before calling the function.</param>
+/// <param name="fn">Function called by the scheduler.</param>
+/// <param name="scheduler"></param>
+let scheduleOnce (after: TimeSpan) (fn: unit -> unit) (scheduler: Scheduler): Async<unit> =
+    let action = Action fn
+    Async.AwaitTask (scheduler.ScheduleOnce(after, action).ContinueWith taskContinuation)
+
+/// <summary>
+/// Schedules a <paramref name="message"/> to be sent to the provided <paramref name="receiver"/> in specified time intervals.
+/// </summary>
+/// <param name="after">Initial delay to first function call.</param>
+/// <param name="every">Interval.</param>
+/// <param name="message">Message to be sent to the receiver by the scheduler.</param>
+/// <param name="receiver">Message receiver.</param>
+/// <param name="scheduler"></param>
+let scheduleTell (after: TimeSpan) (every: TimeSpan) (message: 'Message) (receiver: ActorRef) (scheduler: Scheduler): Async<unit> =
```

Comment:
I like the fact that in the F# API the different schedule operations have different names, rather than lots of overloading